chords/settings/base.py

```diff
@@ -101,7 +101,7 @@
 STATICFILES_FINDERS = [
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
-    'sass_processor.finders.CssFinder'
+    'sass_processor.finders.CssFinder',
```

Comment:
Did this cause a problem?

Reply:
Its not functional, it was there before, I can drop it it